src/IECore/Camera.cpp

```diff
@@ -303,15 +303,14 @@ void Camera::addStandardParameters()
 	}
 
 	// clipping planes
-	V2f clippingPlanes( -1.0f );
+	V2f clippingPlanes( 0.01f, 100000.0f );
```

Comment:
It seems like this could be a lot more concise. Basically the new logic is "if there are no clipping planes, or the type is wrong, then add clipping planes of (0.01, 100000)", so we could totally drop the `clippingPlanes` variable and one branch of the conditional.